yt/frontends/stream/data_structures.py

```diff
@@ -72,6 +76,27 @@ def Children(self):
         return [self.index.grids[cid - self._id_offset] for cid in self._children_ids]
 
 
+class StreamStretchedGrid(StretchedGrid):
+    _id_offset = 0
+
+    def __init__(self, id, index):
+        cell_widths = index.grid_cell_widths[id - self._id_offset]
+        super().__init__(id, cell_widths, None, index)
```

Comment:
seems clearer to omit an argument with a default value rather than passing a `None` positionally
```suggestion
        super().__init__(id, cell_widths, index=index)
```